LICENSES/fetch-licenses.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -u
+#
+# Requires license-detector from https://github.com/go-enry/go-license-detector/releases/tag/v4.3.0 
+# to be installed in the current directory.
```

Comment:
I'd also add 'xml2' here for a complete list of dependencies that need to be preinstalled.

Reply:
I added a whole list of dependencies.